README.md

```diff
@@ -847,6 +847,95 @@ You will need to inject the CSS and Javascript into your views as follows:
 
 References within your code to the Javascript object `accessibleAutocomplete` should be replaced with `HMRCAccessibleAutocomplete`.
 
+### Transforming a Select element into an Accessible Autocomplete element
+
+With the above accessible-autocomplete CSS and Javascript, you can transform your Select element into an
+AccessibleAutocomplete element using the `asAccessibleAutocomplete` implicit helper method on a `Select`, as follows.
+
+```scala
+@import uk.gov.hmrc.govukfrontend.views.Implicits.RichSelect
+
+@govukSelect(Select(...).asAccessibleAutocomplete())
+```
+This will set up an accessible autocomplete element with default values set as below
+```text
+defaultValue = ""
+showAllValues = false
+autoselect = false
+```
+More information on these values can be found [here](https://www.npmjs.com/package/accessible-autocomplete) under the `API documentation` heading
+
+If you wish to change these values you can do so by providing an `AccessibleAutocomplete` model into the `asAccessibleAutocomplete` method as seen below
+```scala
+@govukSelect(Select(...).asAccessibleAutocomplete(Some(
+  AccessibleAutocomplete(
+    defaultValue = Some("United Kingdom"),
```

Comment:
one thing I thought, is I bet this is so that you can have an initial "placeholder text" that makes sense when it's a normal select "select a country", then when it's enhanced you might want to adjust it to "start typing to select a country" 🤷‍♂️ just `defaultValue=` attribute name for me doesn't convey that kind of thing at all

Reply:
yeah, it seems this is really `defaultText`, but i guess we're just mirroring the alphagov API

Reply:
It is misleading, should we change the AccessibleAutocomplete model to have defaultText instead or promptText?

Reply:
@kyle-bowden maybe rather than "United Kingdom" in the example we could put "Start typing to select a country" and in the bit below have "Placeholder text" be "Select an option" - copy probably not quite inline with GDS guidance but makes it clear that this is all around placeholder type text

Reply:
or actually, I'm dithering, looking at the guidance that's not how they use it... lets be consistent with GDS